#error: 8:VARIABLE_NOT_SET

import testsupport.TestException;

val a as int;
try {
	throw new TestException("test");
} catch {
	a = 1;
}

println(a);
